tests/php/Integration/Run/Command/Repl/ReplCommandTest.php

```diff
@@ -167,4 +159,46 @@ private function createReplTestIo(): ReplTestIo
 
         return new ReplTestIo($exceptionPrinter);
     }
+
+    private function prepareRunDependencyProvider(ReplCommandIoInterface $io): void
+    {
+        AbstractClassResolver::resetCache();
```

Comment:
This is not implemented in Gacela yet. We got some new ideas to make this possible, one of them is creating this function. More details will come soon 🤣

Reply:
PR in Gacela ready to be merged: https://github.com/gacela-project/gacela/pull/157